map_model/src/pathfind/ch.rs

```diff
@@ -53,17 +53,16 @@ impl ContractionHierarchyPathfinder {
         }
     }
 
-    pub fn pathfind(&self, req: PathRequest, map: &Map) -> Option<Path> {
-        (match req.constraints {
+    pub fn pathfind(&self, req: PathRequest, map: &Map) -> Option<PathV2> {
+        match req.constraints {
             PathConstraints::Pedestrian => self.walking_graph.pathfind(req, map),
             PathConstraints::Car => self.car_graph.pathfind(req, map),
             PathConstraints::Bike => self.bike_graph.pathfind(req, map),
             PathConstraints::Bus => self.bus_graph.pathfind(req, map),
             // Light rail networks are absolutely tiny; using a contraction hierarchy for them is
             // overkill. And in fact, it costs a bit of memory and file size, so don't do it!
             PathConstraints::Train => dijkstra::pathfind(req, map.routing_params(), map),
-        })
-        .map(|(path, _)| path)
```

Comment:
Lots of places were internally returning (a path, the total pathfinding cost of it). PathV2 just stores that cost directly, since it's pretty useful in many places.